docs/software/How-to-install-Irishub.md

```diff
@@ -8,6 +8,19 @@ Please replace <latest_iris_version> with v0.13.1 while using "git checkout"
 
 You can download the source code from github and compile it locally.
 
+#### Configure Your Server
+
+All the blockchains in IRISnet are based on Cosmos-SDK, which is a framework for building blockchain applications in Golang. It is being used to build [Cosmos Hub](https://cosmos.network/). It's recommended to run a validator node on Linux server.
+
```

Comment:
Start this paragraph with "It's recommended that you run a validator node on Linux Server." and remove the two sentences before it.

Reply:
fixed